Logging operator reports CrashLoopBackOff when I try the steps from README to install the operator

**Describe the bug**

$ kubectl get pods
NAME                                READY   STATUS             RESTARTS   AGE
logging-operator-795f4f66d4-pr4tb   0/1     CrashLoopBackOff   5          5m33s

kubectl logs logging-operator-795f4f66d4-pr4tb
/usr/local/go/src/runtime/asm_amd64.s:1333
panic: assignment to entry in nil map [recovered]
	panic: assignment to entry in nil map
{"level":"info","ts":1551810833.3330257,"logger":"controller_plugin","msg":"Reconciling Plugin","Request.Namespace":"default","Request.Name":"nginx-logging"}
time="2019-03-05T18:33:53Z" level=info msg="Applying filter"
E0305 18:33:53.434889       1 runtime.go:69] Observed a panic: "assignment to entry in nil map" (assignment to entry in nil map)

**To Reproduce**
Steps to reproduce the behavior:
Follow the exact steps of readme for logging operator.

**Expected behavior**
Operator pod
Fluentbit and Fluentd daemon sets should have been created
kubectl get pods should have returned all the created resources.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Additional context**
Add any other context about the problem here.

--- a/README.md
+++ b/README.md
@@ -23,17 +23,17 @@ kubectl create -f deploy/crds/logging_v1alpha1_fluentd_crd.yaml
 
 # If RBAC enabled create the required resources
 kubectl create -f deploy/clusterrole.yaml 
-kubectl create -f deploy/clusterrole_bindig.yaml
+kubectl create -f deploy/clusterrole_binding.yaml
 kubectl create -f deploy/service_account.yaml
 
 # Create the Operator
 kubectl create -f deploy/operator.yaml
 
 # Create the fluent-bit daemonset by submiting a fluent-bit CR
-kubectl create -f deploy/crd/logging_v1alpha1_fluentbit_cr.yaml
+kubectl create -f deploy/crds/logging_v1alpha1_fluentbit_cr.yaml
 
 # Create the fluentd deployment by submitting a fluentd CR
-kubectl create -f deploy/crd/logging_v1alpha1_fluentd_cr.yaml
+kubectl create -f deploy/crds/logging_v1alpha1_fluentd_cr.yaml
 
 ```
 
@@ -71,6 +71,8 @@ apiVersion: "logging.banzaicloud.com/v1alpha1"
 kind: "Plugin"
 metadata:
   name: "nginx-logging"
+  labels:
+    release: test
 spec:
   input:
     label:

